<h2>What is the AVrC ?</h2>
<p> Despite the importance of the gut virome in human health and disease, identifying viral sequences from metagenomic datasets remains computationally challenging. Up to 99% of viral reads lack significant alignments to known viral genomes due to underrepresentation in reference databases. Recent machine learning tools can detect novel viral sequences based on features like k-mer composition or genomic signatures, but are limited to classifying assembled contigs into simplistic viral/non-viral categories. Several large-scale efforts have mined human gut metagenomes to establish viral catalogues, including the Gut Virome Database (33,242 viral OTUs), Cenote Human Virome Database (45,033 OTUs), and Gut Phage Database (142,809 OTUs). However, these catalogues have not been consistently compared for quality, diversity, and completeness. There is an unmet need to harmonize available gut viral sequences into a unified resource for comparing novel viruses against previous efforts. The Aggregated Gut Viral Catalog (AVrC) addresses this gap by harmonizing and aggregating previous mining efforts into a comprehensive resource to allow for the exploration of the Human gut viral diversity and the easier comparison of newly discovered viral sequences. 
<br><br>
You want to get involved in our project? For more information, please contact us at: alise.ponsero[at]qib.ac.edu
<br><br>

<h2>Datasets and catalogues included</h2>
<p>The AVrC harmonize and aggregate viral sequences from previously published and unpublished mining efforts. Below is the list of the ressources included in the current database version:</p>

<ul>
  <li>IMG/Vr (Camargo, Nayfach, et al. 2023) - Human gut subset downloaded on 01.12.24</li>
  <li>Gut virome database (GVD) (Gregory et al. 2020) </li>
  <li>Gut Phage database (GPD) (Camarillo-Guerrero et al. 2021)</li> 
  <li>Metagenomic gut virus (MGV) (Nayfach, PÃ¡ez-Espino, et al. 2021) </li>
  <li>Cenote human virome database (CHVD) (Tisza and Buck 2021) - Subset of viral sequnces marked as "Gut"</li>
  <li>COPSAC infant phages (Shah et al. 2023) </li>
  <li>Gut Phages (KGP) (Benler et al. 2021) </li>
  <li>Japanese 4D catalogue (Nishijima et al. 2022) </li>
  <li>Danish Enteric Virome Catalog (DEVoC) (Van Espen et al. 2021) </li>
</ul>
<br>
<p>Additionally, 12 large-scale infant studies (> 7,000 samples) were screened and the putative viral sequences included in the catalogue (PRJNA345144, PRJEB32135, PRJEB6456, PRJNA384716, PRJNA473126, PRJNA290380, PRJEB42363, PRJNA695570, PRJEB32631, PRJNA497734, PRJNA489090, PRJEB70237)</p>

<br><br>
<h2>Viral Metadata Description:</h2>
<p>The AVrC Web-interface allows users to filter and download a subset of the vOTU reference sequences based on the sequence quality, viral predicted lifestyle, viral and host taxonomy. After selection, the user can download the viral sequence metadata summary: </p>

  <li>contig_id = Unique identifier of all viral sequences in the AVrC database</li>
  <li>vOTU_ID = AVrC species-level viral OTU unique ID</li>
  <li>checkv_quality = Quality category of the sequence, as defined by CheckV</li>
  <li>contig_length = length of the viral sequence</li>
  <li>plasmid_score = Plasmid score classification as defined by GeNomad</li>
  <li>Plasmid = The sequence was classified as a potential plasmid sequence by GeNomad</li>
  <li>pred_lifestyle = Viral lifestyle predicted by PhaTyp, CheckV and GeNomad (see below)</li>
  <li>Realm,Kingdom,Phylum,Class,Order,Family = Viral taxonomy predicted by GeNomad</li>
  <li>Host_Domain,Host_Phylum,Host_Class,Host_Order,Host_Family,Host_Genus = Taxonomic classification of the predicted hosts obtained by iPhop</li>
</ul>
<br>
<p>Note : The putative viral lifestyle strategy was determined using PhaTyp as well as the annotations derived from CheckV and geNomad. Briefly, four categories were generated: temperate (PhaTyp classifies the sequence as temperate with a score of >= 0.7 or CheckV and geNomad predict a prophage sequence), uncertain temperate (PhaTyp classifies the sequence as temperate with a score of < 0.7), virulent (PhaTyp classifies the sequence as virulent with a score of >= 0.7) and uncertain virulent (PhaTyp classifies the sequence as virulent with a score of < 0.7).</p>

<br><br>
<h2>How to cite the AVrC ?</h2>
<p>Example Citation, 2023</p>

